tests/conftest.py

```diff
@@ -221,11 +229,32 @@ def app_config(app_config):
     # Enable communities while in preview
     app_config['COMMUNITIES_ENABLED'] = True
 
-    # Enable DOI miting
+    # Enable DOI minting...
     app_config['DATACITE_ENABLED'] = True
     app_config['DATACITE_USERNAME'] = 'INVALID'
     app_config['DATACITE_PASSWORD'] = 'INVALID'
     app_config['DATACITE_PREFIX'] = '10.1234'
```

Comment:
just out of curiosity: this configuration wouldn't have worked ever anyway, right?
so previously, the tests were sending requests to datacite's test fabrica that would get rejected because of invalid credentials?
but this wouldn't have an impact on the local DOI minting workflow?

Reply:
> tests were sending requests to datacite's test fabrica that would get rejected because of invalid credentials?

Exactly. And since we don't seem to care about the feedback from DataCite, this is a silent error.
It wouldn't impact "local" minting because we mint before we send the request to DataCite.